.github/workflows/integration_tests.yml

```diff
@@ -108,6 +118,10 @@ jobs:
       - name: Install Cloud SDK for mobile integration tests
         if: matrix.target_platform != 'Desktop' && !cancelled()
         uses: GoogleCloudPlatform/github-actions/setup-gcloud@master 
+      - name: Install Google Cloud beta components for iOS integration tests
+        if: matrix.target_platform == 'iOS'
+        run: |
+          gcloud --quiet components install beta
```

Comment:
Is there any harm in putting this command in test_lab.py? This will be needed both locally and on GHA, so it seems sensible to put it in the script instead.

Reply:
There is no harm putting it in test_lab.py, except it require more code and code looks not good(an extra variable to make it only for iOS) in test_lab.py.
To run it locally, there is interactive prompts. Just input "Y" to install the components.

Reply:
Having to include this step in every workflow that uses the script is a significant downside. 

To avoid having to add another flag, the logic can choose to run this command if we find any iOS testapps, i.e. could set a boolean here:
https://github.com/firebase/firebase-cpp-sdk/blob/dev/scripts/gha/test_lab.py#L119

Reply:
This is what I mean the code looks not good. It is because we build iOS and then we got IPAs. It's strange that we say we have IPAs, thus we are building for iOS (especially in a loop).
1. To write it into the script, I prefer adding a flag to test_lab.py itself. Like: --p ${{ matrix.target_platform }}
2. Or we could put it into step "Install Cloud SDK for mobile integration tests":
`      - name: Install Cloud SDK for mobile integration tests
        if: matrix.target_platform != 'Desktop' && !cancelled()
        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master 
        run: |
          gcloud --quiet components install beta
`
   Thus, this step installs both gcloud and gcloud compoent beta. Though it installs gcloud compoent beta for Android, it only takes about 30 seconds.

Reply:
The logic is "we passed a directory with testapps to this tool; one or more of these testapps is for iOS; thus we need to install tools for testing iOS". That's intuitive, no? 

The flag would add complexity to its usage, and is redundant since we need to install these tools if and only if the directory of testapps include iOS testapps, information we already have available to us in the context of the tool.